<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Puerto &amp;serie:</string>
      </property>
      <property name="buddy" >
       <cstring>puerto</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QComboBox" name="puerto" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2" >
     <widget class="QPushButton" name="botonAbrirPuerto" >
      <property name="text" >
       <string>&amp;Abrir puerto</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3" >
     <widget class="QProgressBar" name="feedbackBar" >
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
      <property name="maximum" >
       <number>1</number>
      </property>
      <property name="value" >
       <number>1</number>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="format" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3" >
     <widget class="QFrame" name="framePadre" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Informaci√≥n dispositivo</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QPushButton" name="botonTestNodo" >
            <property name="text" >
             <string>&amp;Probar nodo</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2" >
           <widget class="QLineEdit" name="resultadoTest" >
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QPushButton" name="botonVersion" >
            <property name="text" >
             <string>&amp;Version firmware</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="version0" >
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="buttonSymbols" >
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum" >
             <number>-1</number>
            </property>
            <property name="maximum" >
             <number>9999999</number>
            </property>
            <property name="value" >
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QSpinBox" name="version1" >
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="buttonSymbols" >
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum" >
             <number>-1</number>
            </property>
            <property name="maximum" >
             <number>9999999</number>
            </property>
            <property name="value" >
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Lectura de tarjetas</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="1" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>&amp;Hexadecimal</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buddy" >
             <cstring>leidoHex</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Deci&amp;mal</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buddy" >
             <cstring>leidoDec</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QPushButton" name="botonLeerTarjeta" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&amp;Leer tarjeta</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="leidoHex" >
            <property name="text" >
             <string/>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLineEdit" name="leidoDec" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Escritura de tarjetas</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" colspan="3" >
           <widget class="QLabel" name="wrongData2Write" >
            <property name="minimumSize" >
             <size>
              <width>238</width>
              <height>0</height>
             </size>
            </property>
            <property name="palette" >
             <palette>
              <active>
               <colorrole role="WindowText" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>233</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="WindowText" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>233</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="WindowText" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>119</red>
                  <green>119</green>
                  <blue>121</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="text" >
             <string>Introduzca 10 digitos hexadecimales</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item rowspan="3" row="0" column="3" >
           <widget class="QGroupBox" name="groupBox_4" >
            <property name="title" >
             <string>Opciones</string>
            </property>
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>&amp;Tipo de tarjeta</string>
               </property>
               <property name="buddy" >
                <cstring>tipoDeTarjeta</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QComboBox" name="tipoDeTarjeta" >
               <item>
                <property name="text" >
                 <string>V0</string>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>V1</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0" >
              <widget class="QCheckBox" name="bloquearTarjeta" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>&amp;Bloquear tarjeta</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&amp;Datos</string>
            </property>
            <property name="buddy" >
             <cstring>paraEscribir</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2" >
           <widget class="QLineEdit" name="paraEscribir" >
            <property name="maxLength" >
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2" >
           <widget class="QPushButton" name="botonEscribirTarjeta" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&amp;Escribir tarjeta</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <tabstops>
  <tabstop>puerto</tabstop>
  <tabstop>botonAbrirPuerto</tabstop>
  <tabstop>botonTestNodo</tabstop>
  <tabstop>botonVersion</tabstop>
  <tabstop>version0</tabstop>
  <tabstop>version1</tabstop>
  <tabstop>tipoDeTarjeta</tabstop>
  <tabstop>botonLeerTarjeta</tabstop>
  <tabstop>leidoHex</tabstop>
  <tabstop>leidoDec</tabstop>
  <tabstop>paraEscribir</tabstop>
  <tabstop>bloquearTarjeta</tabstop>
  <tabstop>botonEscribirTarjeta</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
